applications/Chat/coati/models/loss.py

```diff
@@ -65,7 +65,7 @@ def forward(self,
         surr2 = (values - reward)**2
         loss = torch.max(surr1, surr2)
         loss = loss.mean()
-        return loss
```

Comment:
Why to return 0.5*loss ?

Reply:
I hope to follow  a common implementation before value's MSE loss, as shown in